### PR TITLE
Add patient creation events and subscribers

### PR DESCRIPTION
**Pull request by drizzentic**
Add capability to create patient in ADT system from the AMRS flat ADT data.

### CODE REVIEW

File: app/interfaces/patient.ts [hunk 1]
```diff
@@ -11,9 +11,12 @@ declare namespace Patient {
   }
 
   export interface IPatient {
```
Comment (donaldkibet) [inline app/interfaces/patient.ts:13]:
```suggestion
  export interface Patient {
```

File: app/models/patient.ts [hunk 2]
```diff
@@ -6,18 +6,18 @@ export async function loadPatientDataByID(
   connection: Connection
 ) {
   const personCCC = await fetchPersonCCCByID(personId, connection);
-  console.log("ccc", personCCC);
-  return await loadPatientData(personCCC, connection);
+  console.log("ccc");
+  return await loadPatientData(personCCC[0].patient_ccc_number, connection);
```
Comment (donaldkibet) [inline app/models/patient.ts:10]: What happens when `personCCC[0]` is empty

File: app/subscribers/patient.ts [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+  public onPatientCreate({ patient, mflcode }: any) {
+    console.log("SDSD", mflcode, patient);
+    let patients: Patient.IPatient[] = patient;
+    patients[0].patient_number = patients[0].patient_ccc_number;
+    patients[0].mfl_code = mflcode;
+    // Only active patient are eligible for dispense
+    patients[0].current_status = 1;
+    patients[0].partner_status = "Unknown";
+    let payload = patients[0];
+    console.log(payload);
```
Comment (donaldkibet) [inline app/subscribers/patient.ts:69]:
```suggestion
    
```

**Review by donaldkibet — APPROVED** [review]
lgtm with few suggestions

File: app/models/patient.ts (hunk 2 above)
Comment (drizzentic) [reply app/models/patient.ts:10]: I Will put checks in place to handle that. Thanks

**Pull request merged**